uaclient/security.py

```diff
@@ -410,6 +420,8 @@ def prompt_for_affected_packages(
     )
     msg_suffix = ": " + ", ".join(affected_pkg_status.keys())
     msgs = [msg + msg_suffix]
+    upgrade_packages = []
+    cloud_type = get_cloud_type()
```

Comment:
It seems that we are not using this variable ? Also, why would we need the cloud_type in this function ?

Reply:
Ahh this should be a followup PR. I accidentally left this in, it will be used later. I'll back it out of this PR